docs/en/connector/sink/Assert.md

```diff
@@ -0,0 +1,105 @@
+# Assert
+
+> # Sink plugin: Assert [Flink]
+
+## Description
+
+A flink sink plugin which can assert illegal data by user defined rules
+
+:::tip
+
+Engine Supported and plugin name
+
+* [ ] Spark
+* [x] Flink: AssertSink
+
+:::
+
+## Options
+
+| name                    | type     | required | default value |
+| ----------------------- | -------- | -------- | ------------- |
+| rules         		  | ConfigList | yes      | -             |
+| &ensp;field_name        | `String` | yes       | -     |
```

Comment:
remove unknown `&ensp;` string

Reply:
` &ensp;`  is for the indentation because field_name is sub property of rules .  So what's proper substitute of ` &ensp;`.

Reply:
> ` &ensp;` is for the indentation because field_name is sub property of rules . So what's proper substitute of ` &ensp;`.

how about `rules.field_name`?